src/researchhub_access_group/constants.py

```diff
@@ -1,5 +1,8 @@
 ADMIN = "ADMIN"
 EDITOR = "EDITOR"
+ASSISTANT_EDITOR = "ASSISTANT_EDITOR"
```

Comment:
Are we able to display the editor status on profile page? Can you recommend an approach for us to do it? 

e.g Leo sun is Associate Editor of Neurology